### PR TITLE
Fixed root trace for CLI instrumentation

### PR DESCRIPTION
**Pull request by gechetspr**
## PR Description

TBD

## Checklist
- [ ] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: src/Spryker/Service/OtelConsoleInstrumentation/OpenTelemetry/ConsoleInstrumentation.php [hunk 1]
```diff
@@ -97,14 +111,71 @@ function: static::METHOD_NAME,
...
+     * @param string $application
+     * @param \Spryker\Shared\Opentelemetry\Request\RequestProcessor $requestProcessor
+     *
+     * @return void
+     */
+    protected static function registerBootstrapHook(string $className, string $application, RequestProcessor $requestProcessor): void
+    {
+        hook(
+            class: $className,
+            function: '__construct',
```
Comment (olhalivitchuk) [inline src/Spryker/Service/OtelConsoleInstrumentation/OpenTelemetry/ConsoleInstrumentation.php:135]: let's move to const

File: src/Spryker/Service/OtelConsoleInstrumentation/OpenTelemetry/ConsoleInstrumentation.php [hunk 2]
```diff
@@ -97,14 +111,71 @@ function: static::METHOD_NAME,
...
+            pre: static function ($instance, array $params, string $class, string $function, ?string $filename, ?int $lineno) use ($requestProcessor, $application): void {
+                putenv('OTEL_SERVICE_NAME=' . $application);
```
Comment (olhalivitchuk) [inline src/Spryker/Service/OtelConsoleInstrumentation/OpenTelemetry/ConsoleInstrumentation.php:137]: was it well tested? last time when I tried to put it this way there was a total mess in Grafana after sending requests from different endpoint

File: src/Spryker/Service/OtelConsoleInstrumentation/OpenTelemetry/ConsoleInstrumentation.php [hunk 3]
```diff
@@ -97,14 +111,71 @@ function: static::METHOD_NAME,
...
+                if (!defined('OTEL_CLI_TRACE_ID')) {
+                    define('OTEL_CLI_TRACE_ID', uuid_create());
+                }
+
+                $input = [static::CLI_TRACE_ID => OTEL_CLI_TRACE_ID];
+                TraceContextPropagator::getInstance()->inject($input);
+
+                $span = $instrumentation
+                    ->tracer()
+                    ->spanBuilder(static::formatSpanName($requestProcessor->getRequest()))
```
Comment (olhalivitchuk) [inline src/Spryker/Service/OtelConsoleInstrumentation/OpenTelemetry/ConsoleInstrumentation.php:152]: Probably make it the same as in `register()` 
```suggestion
                    ->spanBuilder('Run: '  . static::formatSpanName($requestProcessor->getRequest()))
```

**Review by olhalivitchuk — CHANGES_REQUESTED** [review]

File: src/Spryker/Service/OtelConsoleInstrumentation/OpenTelemetry/ConsoleInstrumentation.php (hunk 2 above)
Comment (gechetspr) [reply src/Spryker/Service/OtelConsoleInstrumentation/OpenTelemetry/ConsoleInstrumentation.php:137]: I had no mess in my testing. So I have all application defined in the same way and for now all looks good.

**Review by olhalivitchuk — APPROVED** [review]

**Pull request merged**